src/components/details-panel/_macro-options.md

```diff
@@ -0,0 +1,22 @@
+| Name   | Type           | Required | Description                                             |
+| ------ | -------------- | -------- | ------------------------------------------------------- |
+| id     | string         | true     | HTML `id` attribute for the details panel               |
+| title  | string         | true     | Title of the component                                  |
+| groups | array`<group>` | true     | Settings for the array of [group items](#group)         |
+| open   | boolean        | false    | Forces the details panel to be open when the page loads |
+
+## Group
```

Comment:
We no longer have multiple groups so I think we can rename this `detailsItems` or something